proto/ddl_service.proto

```diff
@@ -138,12 +139,19 @@ message DropViewResponse {
   uint64 version = 2;
 }
 
+enum TableJobType {
```

Comment:
What other variants are expected in the future? Could be please add some docs for it?

Reply:
May also leave these comments in the source files.